source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/TokenFragment.java

```diff
@@ -301,7 +314,8 @@ public void onResult(
                         }
                 );
             } else {
-                Log.d(TAG,"No connected Games API, waiting for onConnected");
+                Log.d(TAG,"No connected Games API");
+                onSignedIn(CommonStatusCodes.ERROR, null);
```

Comment:
not sure about here if the pendingTokenRequest might need to get cancelled as well

Reply:
Thanks again, I'll take a look at these today.